### PR TITLE
Add help about Gentoo/Linux

### PR DESCRIPTION
**Pull request by Freed-Wu**
Add help about Gentoo/Linux

### CODE REVIEW

File: docs/_docs/installation/other-linux.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ On Debian:
 sudo apt-get install ruby-full build-essential
 ```
 
+On Gentoo/Linux:
```
Comment (zdroid) [inline docs/_docs/installation/other-linux.md]:
```suggestion
On Gentoo Linux:
```
Shouldn't the distro name be stylized this way?

**Comment by chrisfinazzo** [comment]
I _think_ so, at least according to Gentoo’s website. As to the substance of this change, I can’t speak intelligently about Linux, so I’ll let someone else make the call.

**Comment by Freed-Wu** [comment]
2020-02-13 09-45-45 的屏幕截图

2020-02-13 09-56-05 的屏幕截图

**Comment by mattr-** [comment]
@jekyllbot: merge +docs

**Pull request merged**